sidekick_core/lib/src/commands/format_command.dart

```diff
@@ -0,0 +1,158 @@
+      help:
+          'Only verifies that all code is formatted, does not actually format it',
+    );
+  }
+
+  @override
+  Future<void> run() async {
+    final String? packageName = argResults?['package'] as String?;
+    final int? lineLength =
+        int.tryParse(argResults?['line-length'] as String? ?? '');
+    final bool verify = argResults?['verify'] as bool? ?? false;
+
+    final root = SidekickContext.projectRoot;
+    final List<DartPackage> allPackages = findAllPackages(root);
+    final globExcludes =
+        excludeGlob.map<Glob>((rule) => Glob("${root.path}/$rule"));
+    if (packageName != null) {
+      final package =
+          allPackages.where((it) => it.name == packageName).firstOrNull;
+      if (package == null) {
+        throw "Package with name $packageName not found in repository "
+            "${SidekickContext.repository?.path}";
+      }
+      final int lineLength = getLineLength(package);
+      final allDartFiles =
+          package.root.listSync(recursive: true).filterAllFiles(
+                globExcludes,
+              );
+      _format({lineLength: allDartFiles}, verify: verify);
+      return;
+    }
+
+    // Getting all Dart files exluding files which are starting with a .
+    final allFiles = SidekickContext.projectRoot
+        .listSync(recursive: true)
+        .filterAllFiles(
+          globExcludes,
+        )
+        .toList();
+
+    // Getting all directories excluding directories which are starting with a . and sort them by length
+    final sortedPackages =
+        allPackages.sortedByDescending((element) => element.root.path.length);
+
+    final lineLengthsAndFiles = <int, List<File>>{};
+
+    for (final package in sortedPackages) {
+      final resolvedLineLength = lineLength ?? getLineLength(package);
+      final filesInPackage = allFiles
+          .where((file) => file.path.contains(package.root.path))
+          .toList();
+      allFiles.removeWhere((file) => filesInPackage.contains(file));
+      if (filesInPackage.isNotEmpty) {
+        (lineLengthsAndFiles[resolvedLineLength] ??= []).addAll(filesInPackage);
+      }
+    }
+    _format(lineLengthsAndFiles, verify: verify);
+  }
+}
+
+extension on Iterable<FileSystemEntity> {
+  Iterable<File> filterAllFiles(Iterable<Glob> globExcludes) {
+    return whereType<File>()
+        .filter((file) => file.extension == '.dart')
+        .filter(
+          (file) => file.uri.pathSegments.none(
+            (element) => element.startsWith('.'),
+          ),
+        )
+        .whereNot(
+          (file) => globExcludes.any(
+            (glob) => glob.matches(file.path),
+          ),
+        );
+  }
+}
+
+void _format(
+  Map<int, Iterable<File>> filesWithLineLength, {
+  bool verify = false,
+}) {
+  // remove map entries with 0 files, otherwise the `format` command crashes
+  // because it expects at least one file or directory
+  for (final entry in filesWithLineLength.entries) {
+    final exitCode = dart(
```

Comment:
When `format --verify` fails it prints an ugly error message. You have to use `nothrow: true`

```
Unhandled exception:
/opt/homebrew/Cellar/dart/2.19.0/libexec/bin/dart format -l 80 /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/templates/root_with_packages/packages/package_b/lib/package_b.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/templates/root_with_packages/packages/package_a/lib/package_a.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/templates/minimal_flutter_package/lib/minimal.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/templates/minimal_sidekick_plugin/lib/minimal_sidekick_plugin.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/templates/minimal_sidekick_plugin/tool/install.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/templates/minimal_dart_package/lib/minimal.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/templates/root_with_packages/lib/root_with_packages.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_plugin_installer/test/add_dependency_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_plugin_installer/lib/sidekick_plugin_installer.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_plugin_installer/lib/src/pub_get.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_plugin_installer/lib/src/register_plugin.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_plugin_installer/lib/src/add_dependency.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_plugin_installer/lib/src/add_import.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_plugin_installer/lib/src/pub_add_dependency.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_plugin_installer/lib/src/plugin_context.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_plugin_installer/lib/src/pub_add_local_dependency.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/test/delete_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/test/vault_command_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/test/unlock_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/example/example.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/lib/sidekick_vault.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/lib/src/encrypted_vault_string.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/lib/src/gpg.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/lib/src/commands/vault_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/lib/src/logging.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/lib/src/vault.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/tool/install.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_test/lib/fake_stdio.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_test/lib/file_creation_observer.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_test/lib/sidekick_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_test/lib/src/fake_stdio.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_test/lib/src/local_testing.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_test/lib/src/delegated_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_test/lib/src/fake_sdk.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/deps_command_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/migration_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/update/patches/157_fix_usage_message.patch_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/update/patches/208_remove_cli_name.patch_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/update/update_dart_version_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/flutter_command_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/version_checker_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/analyzer_command_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/flutter_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/init_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/sidekick_package_template_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/sidekick_command_runner_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/dart_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/format_command_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/dart_command_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/dart_package.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/update_command_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/bash_command_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/sidekick_core.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/directory_ext.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/sidekick_package.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/cli_util.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/update/patches/patch_migrations.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/update/patches/208_remove_cli_name.patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/update/patches/157_fix_usage_message.patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/update/migration.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/dart_runtime.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/sidekick_context.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/template/entrypoint.template.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/template/install.sh.template.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/template/sidekick_config.sh.template.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/template/download_dart.sh.template.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/template/sidekick_package.template.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/template/run.sh.template.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/dart.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/repository.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/forward_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/version_checker.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/update_sidekick_cli.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/dart_package.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/sidekick_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/flutter_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/recompile_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/deps_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/plugins/install_plugin_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/plugins/create_templates/plugin_template_generator.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/plugins/create_templates/install_only.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/plugins/create_templates/shared_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/plugins/create_templates/shared_code.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/plugins/create_plugin_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/plugins/plugins_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/update_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/install_global_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/analyze_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/dart_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/bash_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/format_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/pub/pub.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/file_util.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/global_sidekick_root.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/flutter.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/test/sidekick_context_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/bin/main.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/lib/sk_sidekick.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/lib/src/release/sidekick_bundled_version_bump.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/lib/src/release/sidekick_core_bundled_version_bump.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/lib/src/sk_project.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/lib/src/commands/bump_version_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/lib/src/commands/test_sidekick_context_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/lib/src/commands/coverage_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/lib/src/commands/release_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/lib/src/commands/lock_dependencies_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/lib/src/commands/verify_publish_state_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/bin/resources/devtools/assets/assets/scripts/inspector_polyfill_script.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/svg/dart2js/svg_dart2js.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/isolate_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/js_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/internal_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/io_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/core_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/convert_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/collection_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/math_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/developer_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/typed_data_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/string_helper.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/js_names.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/js_string.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/identity_hash_map.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/annotations.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/profile.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/runtime_metrics.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/linked_hash_map.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/foreign_helper.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/mirror_helper.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/isolate_helper.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/js_helper.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/js_rti.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/native_helper.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/rtti.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/utils.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/types.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/classes.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/runtime.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/native_typed_data.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/interceptors.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/regexp_helper.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/debugger.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/js_array.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/js_number.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/custom_hash_map.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/js_primitives.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/string_helper.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/isolate_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/js_names.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/js_string.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/late_helper.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/dart2js_runtime_metrics.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/annotations.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/linked_hash_map.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/constant_map.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/foreign_helper.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/js_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/instantiation.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/synced/async_await_error_codes.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/synced/embedded_names.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/js_helper.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/internal_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/io_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/core_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/convert_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/native_helper.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/native_typed_data.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/collection_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/interceptors.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/regexp_helper.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/js_array.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/async_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/math_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/js_number.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/developer_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/typed_data_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/js_primitives.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/type.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/identical_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/expando_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/growable_list.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/hash_factories.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/stack_trace_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/js_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/js_helper.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/errors_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/class_id.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/list.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/internal_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/timer_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/core_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/int.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/string_buffer_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/print_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/object_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/date_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/js_util_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/double.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/function.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/developer.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/bool.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/math_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/string_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/libraries.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/file_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/directory_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/builtin.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/socket_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/io_service_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/filter_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/process_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/sync_socket_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/common_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/platform_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/vmservice_io.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/cli_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/vmservice_server.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/stdio_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/eventhandler_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/namespace_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/file_system_entity_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/secure_socket_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/finalizer_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/identical_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/isolate_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/compact_hash.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/ffi_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/double_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/empty_source.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/expando_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/timer_impl.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/hash_factories.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/deferred_load_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/stopwatch_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/timeline.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/lib_prefix.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/invocation_mirror_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/errors_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/integers.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/internal_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/ffi_native_type_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/timer_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/core_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/mirror_reference.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/convert_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/array.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/profiler.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/ffi_struct_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/uri_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/function_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/collection_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/class_id_fasta.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/string_buffer_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/array_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/print_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/ffi_dynamic_library_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/integers_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/object_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/growable_array.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/mirrors_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/ffi_allocation_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/weak_property.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/date_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/double.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/bigint_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/symbol_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/map_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/immutable_map.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/regexp_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/function.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/developer.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/type_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/bool_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/schedule_microtask_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/async_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/mirrors_impl.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/math_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/ffi_native_finalizer_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/string_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/typed_data_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/stacktrace.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/null_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/sdk_library_metadata/lib/libraries.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_shared/lib/rti.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_shared/lib/synced/embedded_names.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_shared/lib/synced/recipe_syntax.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_shared/lib/js_util_patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/developer/extension.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/developer/timeline.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/developer/profiler.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/developer/service.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/developer/developer.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/web_audio/dart2js/web_audio_dart2js.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/web_gl/dart2js/web_gl_dart2js.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/iterable.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/collection.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/maps.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/hash_set.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/linked_hash_map.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/queue.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/list.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/splay_tree.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/iterator.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/linked_hash_set.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/set.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/hash_map.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/collections.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/linked_list.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/type.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/duration.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/iterable.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/invocation.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/date_time.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/stopwatch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/annotations.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/bigint.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/weak.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/map.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/comparable.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/core.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/print.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/sink.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/string.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/identical.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/object.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/list.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/pattern.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/int.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/null.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/regexp.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/symbol.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/enum.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/num.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/exceptions.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/double.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/iterator.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/string_sink.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/function.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/set.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/bool.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/errors.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/uri.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/stacktrace.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/string_buffer.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/isolate/isolate.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/isolate/capability.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/js/js_wasm.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/js/_js_client.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/js/_js.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/js/js.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/js/_js_server.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/js/_js_annotations.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/stream_impl.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/stream_pipe.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/async_error.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/async.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/future.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/stream_controller.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/schedule_microtask.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/stream_transformers.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/deferred_load.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/future_impl.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/stream.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/timer.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/broadcast_stream_controller.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/zone.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/namespace_impl.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/link.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/process.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/string_transformer.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/file_impl.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/secure_server_socket.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/io_service.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/embedder_config.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/stdio.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/service_object.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/file_system_entity.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/eventhandler.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/io.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/directory.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/file.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/directory_impl.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/overrides.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/socket.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/common.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/security_context.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/io_resource_info.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/platform.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/sync_socket.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/secure_socket.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/io_sink.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/network_profiling.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/data_transformer.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/platform_impl.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/patch.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/iterable.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/internal.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/sort.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/lowering.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/print.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/async_cast.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/list.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/symbol.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/bytes_builder.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/cast.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/errors.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/linked_list.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/dart2js/html_dart2js.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/dartium/nativewrappers.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/html_common/conversions_dart2js.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/html_common/filtered_element_list.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/html_common/lists.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/html_common/device.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/html_common/metadata.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/html_common/html_common_dart2js.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/html_common/conversions.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/html_common/css_class_set.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/html_common/html_common.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/cli/wait_for.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/cli/cli.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/math/point.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/math/random.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/math/rectangle.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/math/math.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/mirrors/mirrors.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/js_util/js_util_wasm.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/js_util/js_util.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/native_type.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/annotations.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/union.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/abi.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/allocation.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/c_type.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/ffi.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/abi_specific.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/native_finalizer.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/struct.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/dynamic_library.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/http_parser.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/embedder_config.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/http_session.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/websocket.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/overrides.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/http_date.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/http.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/http_headers.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/crypto.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/websocket_impl.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/http_impl.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/typed_data/typed_data.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/typed_data/unmodifiable_typed_data.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/web_sql/dart2js/web_sql_dart2js.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/indexed_db/dart2js/indexed_db_dart2js.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/base64.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/byte_conversion.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/ascii.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/utf.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/json.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/string_conversion.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/latin1.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/html_escape.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/converter.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/convert.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/chunked_conversion.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/line_splitter.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/codec.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/encoding.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/test_runner.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/update_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/recompile_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/dcli_ask_validators_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/util/cli_runner.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/init_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/templates/templates.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/plugins_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/directory_extension_test.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/bin/sidekick.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/lib/sidekick.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/lib/src/util/dcli_ask_validators.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/lib/src/util/name_suggester.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/lib/src/util/directory_extension.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/lib/src/commands/init_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/lib/src/commands/update_command.dart /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/lib/src/commands/plugins_command.dart --set-exit-if-changed 
exit: 65
reason: The command [/opt/homebrew/Cellar/dart/2.19.0/libexec/bin/dart] with args [format, -l, 80, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/templates/root_with_packages/packages/package_b/lib/package_b.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/templates/root_with_packages/packages/package_a/lib/package_a.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/templates/minimal_flutter_package/lib/minimal.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/templates/minimal_sidekick_plugin/lib/minimal_sidekick_plugin.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/templates/minimal_sidekick_plugin/tool/install.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/templates/minimal_dart_package/lib/minimal.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/templates/root_with_packages/lib/root_with_packages.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_plugin_installer/test/add_dependency_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_plugin_installer/lib/sidekick_plugin_installer.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_plugin_installer/lib/src/pub_get.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_plugin_installer/lib/src/register_plugin.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_plugin_installer/lib/src/add_dependency.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_plugin_installer/lib/src/add_import.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_plugin_installer/lib/src/pub_add_dependency.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_plugin_installer/lib/src/plugin_context.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_plugin_installer/lib/src/pub_add_local_dependency.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/test/delete_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/test/vault_command_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/test/unlock_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/example/example.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/lib/sidekick_vault.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/lib/src/encrypted_vault_string.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/lib/src/gpg.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/lib/src/commands/vault_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/lib/src/logging.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/lib/src/vault.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_vault/tool/install.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_test/lib/fake_stdio.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_test/lib/file_creation_observer.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_test/lib/sidekick_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_test/lib/src/fake_stdio.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_test/lib/src/local_testing.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_test/lib/src/delegated_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_test/lib/src/fake_sdk.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/deps_command_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/migration_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/update/patches/157_fix_usage_message.patch_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/update/patches/208_remove_cli_name.patch_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/update/update_dart_version_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/flutter_command_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/version_checker_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/analyzer_command_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/flutter_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/init_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/sidekick_package_template_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/sidekick_command_runner_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/dart_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/format_command_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/dart_command_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/dart_package.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/update_command_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/test/bash_command_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/sidekick_core.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/directory_ext.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/sidekick_package.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/cli_util.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/update/patches/patch_migrations.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/update/patches/208_remove_cli_name.patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/update/patches/157_fix_usage_message.patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/update/migration.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/dart_runtime.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/sidekick_context.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/template/entrypoint.template.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/template/install.sh.template.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/template/sidekick_config.sh.template.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/template/download_dart.sh.template.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/template/sidekick_package.template.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/template/run.sh.template.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/dart.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/repository.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/forward_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/version_checker.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/update_sidekick_cli.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/dart_package.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/sidekick_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/flutter_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/recompile_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/deps_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/plugins/install_plugin_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/plugins/create_templates/plugin_template_generator.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/plugins/create_templates/install_only.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/plugins/create_templates/shared_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/plugins/create_templates/shared_code.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/plugins/create_plugin_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/plugins/plugins_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/update_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/install_global_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/analyze_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/dart_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/bash_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/commands/format_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/pub/pub.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/file_util.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/global_sidekick_root.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick_core/lib/src/flutter.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/test/sidekick_context_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/bin/main.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/lib/sk_sidekick.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/lib/src/release/sidekick_bundled_version_bump.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/lib/src/release/sidekick_core_bundled_version_bump.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/lib/src/sk_project.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/lib/src/commands/bump_version_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/lib/src/commands/test_sidekick_context_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/lib/src/commands/coverage_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/lib/src/commands/release_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/lib/src/commands/lock_dependencies_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/lib/src/commands/verify_publish_state_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/bin/resources/devtools/assets/assets/scripts/inspector_polyfill_script.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/svg/dart2js/svg_dart2js.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/isolate_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/js_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/internal_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/io_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/core_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/convert_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/collection_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/math_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/developer_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/patch/typed_data_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/string_helper.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/js_names.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/js_string.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/identity_hash_map.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/annotations.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/profile.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/runtime_metrics.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/linked_hash_map.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/foreign_helper.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/mirror_helper.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/isolate_helper.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/js_helper.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/js_rti.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/native_helper.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/rtti.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/utils.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/types.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/classes.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/runtime.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/native_typed_data.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/interceptors.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/regexp_helper.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/debugger.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/js_array.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/js_number.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/custom_hash_map.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_dev_runtime/private/js_primitives.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/string_helper.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/isolate_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/js_names.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/js_string.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/late_helper.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/dart2js_runtime_metrics.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/annotations.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/linked_hash_map.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/constant_map.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/foreign_helper.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/js_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/instantiation.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/synced/async_await_error_codes.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/synced/embedded_names.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/js_helper.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/internal_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/io_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/core_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/convert_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/native_helper.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/native_typed_data.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/collection_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/interceptors.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/regexp_helper.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/js_array.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/async_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/math_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/js_number.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/developer_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/typed_data_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_runtime/lib/js_primitives.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/type.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/identical_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/expando_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/growable_list.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/hash_factories.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/stack_trace_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/js_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/js_helper.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/errors_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/class_id.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/list.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/internal_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/timer_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/core_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/int.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/string_buffer_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/print_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/object_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/date_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/js_util_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/double.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/function.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/developer.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/bool.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/math_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/wasm/lib/string_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/libraries.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/file_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/directory_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/builtin.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/socket_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/io_service_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/filter_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/process_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/sync_socket_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/common_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/platform_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/vmservice_io.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/cli_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/vmservice_server.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/stdio_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/eventhandler_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/namespace_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/file_system_entity_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/bin/secure_socket_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/finalizer_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/identical_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/isolate_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/compact_hash.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/ffi_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/double_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/empty_source.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/expando_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/timer_impl.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/hash_factories.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/deferred_load_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/stopwatch_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/timeline.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/lib_prefix.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/invocation_mirror_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/errors_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/integers.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/internal_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/ffi_native_type_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/timer_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/core_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/mirror_reference.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/convert_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/array.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/profiler.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/ffi_struct_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/uri_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/function_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/collection_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/class_id_fasta.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/string_buffer_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/array_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/print_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/ffi_dynamic_library_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/integers_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/object_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/growable_array.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/mirrors_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/ffi_allocation_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/weak_property.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/date_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/double.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/bigint_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/symbol_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/map_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/immutable_map.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/regexp_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/function.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/developer.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/type_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/bool_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/schedule_microtask_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/async_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/mirrors_impl.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/math_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/ffi_native_finalizer_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/string_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/typed_data_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/stacktrace.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/vm/lib/null_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/sdk_library_metadata/lib/libraries.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_shared/lib/rti.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_shared/lib/synced/embedded_names.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_shared/lib/synced/recipe_syntax.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_internal/js_shared/lib/js_util_patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/developer/extension.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/developer/timeline.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/developer/profiler.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/developer/service.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/developer/developer.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/web_audio/dart2js/web_audio_dart2js.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/web_gl/dart2js/web_gl_dart2js.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/iterable.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/collection.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/maps.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/hash_set.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/linked_hash_map.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/queue.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/list.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/splay_tree.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/iterator.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/linked_hash_set.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/set.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/hash_map.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/collections.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/collection/linked_list.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/type.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/duration.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/iterable.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/invocation.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/date_time.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/stopwatch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/annotations.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/bigint.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/weak.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/map.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/comparable.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/core.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/print.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/sink.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/string.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/identical.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/object.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/list.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/pattern.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/int.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/null.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/regexp.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/symbol.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/enum.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/num.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/exceptions.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/double.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/iterator.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/string_sink.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/function.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/set.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/bool.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/errors.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/uri.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/stacktrace.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/core/string_buffer.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/isolate/isolate.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/isolate/capability.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/js/js_wasm.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/js/_js_client.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/js/_js.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/js/js.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/js/_js_server.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/js/_js_annotations.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/stream_impl.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/stream_pipe.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/async_error.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/async.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/future.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/stream_controller.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/schedule_microtask.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/stream_transformers.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/deferred_load.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/future_impl.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/stream.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/timer.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/broadcast_stream_controller.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/async/zone.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/namespace_impl.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/link.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/process.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/string_transformer.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/file_impl.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/secure_server_socket.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/io_service.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/embedder_config.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/stdio.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/service_object.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/file_system_entity.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/eventhandler.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/io.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/directory.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/file.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/directory_impl.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/overrides.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/socket.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/common.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/security_context.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/io_resource_info.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/platform.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/sync_socket.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/secure_socket.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/io_sink.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/network_profiling.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/data_transformer.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/io/platform_impl.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/patch.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/iterable.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/internal.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/sort.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/lowering.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/print.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/async_cast.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/list.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/symbol.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/bytes_builder.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/cast.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/errors.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/internal/linked_list.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/dart2js/html_dart2js.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/dartium/nativewrappers.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/html_common/conversions_dart2js.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/html_common/filtered_element_list.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/html_common/lists.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/html_common/device.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/html_common/metadata.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/html_common/html_common_dart2js.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/html_common/conversions.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/html_common/css_class_set.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/html/html_common/html_common.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/cli/wait_for.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/cli/cli.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/math/point.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/math/random.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/math/rectangle.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/math/math.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/mirrors/mirrors.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/js_util/js_util_wasm.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/js_util/js_util.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/native_type.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/annotations.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/union.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/abi.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/allocation.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/c_type.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/ffi.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/abi_specific.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/native_finalizer.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/struct.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/ffi/dynamic_library.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/http_parser.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/embedder_config.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/http_session.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/websocket.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/overrides.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/http_date.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/http.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/http_headers.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/crypto.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/websocket_impl.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/_http/http_impl.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/typed_data/typed_data.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/typed_data/unmodifiable_typed_data.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/web_sql/dart2js/web_sql_dart2js.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/indexed_db/dart2js/indexed_db_dart2js.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/base64.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/byte_conversion.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/ascii.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/utf.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/json.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/string_conversion.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/latin1.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/html_escape.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/converter.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/convert.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/chunked_conversion.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/line_splitter.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/codec.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sk_sidekick/build/cache/dart-sdk/lib/convert/encoding.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/test_runner.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/update_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/recompile_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/dcli_ask_validators_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/util/cli_runner.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/init_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/templates/templates.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/plugins_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/test/directory_extension_test.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/bin/sidekick.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/lib/sidekick.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/lib/src/util/dcli_ask_validators.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/lib/src/util/name_suggester.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/lib/src/util/directory_extension.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/lib/src/commands/init_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/lib/src/commands/update_command.dart, /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick/lib/src/commands/plugins_command.dart, --set-exit-if-changed] failed with exitCode: 65 workingDirectory: /Users/pascalwelsch/Projects/phntmxyz/sidekick/sidekick
wait_for_ex.dart : waitForEx : 27
runnable_process.dart : RunnableProcess._waitForExit : 342
runnable_process.dart : RunnableProcess.run : 185
run.dart : startFromArgs : 180
dart.dart : dart : 43
format_command.dart : _format : 136
format_command.dart : FormatCommand.run : 108
command_runner.dart : CommandRunner.runCommand : 209
sidekick_core.dart : SidekickCommandRunner.run : 291
sk_sidekick.dart : runSk : 40
main.dart : main : 4
isolate_patch.dart : _delayEntrypointInvocation.<anonymous closure> : 295
isolate_patch.dart : _RawReceivePortImpl._handleMessage : 192
```